website/addons/wiki/static/WikiEditor.js

```diff
@@ -113,10 +115,10 @@ function ViewModel(url, viewText) {
 
     self.wikisDiffer = function(wiki1, wiki2) {
         // Handle inconsistencies in newline notation
-        var clean1 = typeof wiki1 === 'string' ?
-            wiki1.replace(/(\r\n|\n|\r)/gm, '\n') : '';
-        var clean2 = typeof wiki2 === 'string' ?
-            wiki2.replace(/(\r\n|\n|\r)/gm, '\n') : '';
+        var clean1 = typeof wiki1 === 'string' ? 
+         wiki1.replace(/(\r\n|\n|\r)/gm, '\n'):' ';
+         var clean2 = typeof wiki2 === 'string' ? 
+          wiki2.replace(/(\r\n|\n|\r)/gm, '\n'):' ';
```

Comment:
I don't think u need a space for this empty string but u will need a space before and after ":" sign, check the osf code guideline.